htmlconverter/src/commonMain/kotlin/be/digitalia/compose/htmlconverter/internal/AnnotatedStringHtmlHandler.kt

```diff
@@ -108,6 +109,7 @@ internal class AnnotatedStringHtmlHandler(
             "sub" -> handleSpanStyleStart(SpanStyle(baselineShift = BaselineShift.Subscript))
             "h1", "h2", "h3", "h4", "h5", "h6" -> handleHeadingStart(name)
             "script", "head", "table", "form", "fieldset" -> handleSkippedTagStart()
+            "span" -> handleSpanStart(attributes("style").orEmpty())
```

Comment:
Android support CSS colors for `p` and `li` tags as well, we could consider doing the same.